easy_switch_user/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+================
+Easy Switch User
+================
+
+This module lets administrators and developers quickly change user to test e.g. access rights.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Set the debug mode
+#. Go to Settings > Users & Companies > Users
+#. Choose the user
+#. Click on the "Switch User" button at the right of the user name
+
+.. image:: ../easy_switch_user/static/description/switch_user_button.png
+   :alt: Switch User Button
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/250/15.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/server-ux/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Dennis Sluijk <d.sluijk@onestein.nl>
+* Rolando Duarte <rolando@vauxoo.com>
```

Comment:
Please add information to respective sections under the readme directory, use [the sections specified on the template](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme) and then generate the README using [this tool](https://github.com/OCA/maintainer-tools/#readme-generator)

Reply:
I added the readme directory, I think I do not nedd to generate the README file, because it will be generated automatically, right?